region/maridia/inner-pink/Colosseum.json

```diff
@@ -219,6 +219,59 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Frozen Mochtroid, X-Mode Shinecharge, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "canTrickyUseFrozenEnemies",
+        {"or": [
+          "Spazer",
+          "Wave",
+          "Plasma",
+          "canTrickyDodgeEnemies"
+        ]},
```

Comment:
I don't know why these beams are here.
But avoiding mochtroid damage is canInsaneJump

Reply:
I found Ice+Spazer (at least) to be fast enough to clear the first Mochtroid before it can do any damage (accounting for X-Ray cooldown).

Probably could alternatively use a Missile for the CF path.